code/modules/photography/camera/camera.dm

```diff
@@ -218,8 +219,11 @@
 	var/icon/get_icon = camera_get_icon(turfs, target_turf, psize_x, psize_y, clone_area, size_x, size_y, (size_x * 2 + 1), (size_y * 2 + 1))
 	qdel(clone_area)
 	get_icon.Blend("#000", ICON_UNDERLAY)
+	for(var/mob/living/carbon/human/person as anything in mobs)
+		if(person.is_face_visible())
+			real_names.Add("[person.real_name]")
```

Comment:
Should you really be using `real_name` for this and not `name`?
Seems weird

also, prefer to use `+=` rather than `Add`